check_for_new_releases.py

```diff
@@ -43,7 +46,7 @@ def get_repo_url(artifacthub_config) -> str:
 
                 try:
                     repo = g.get_repo(repository_name)
-                except:
+                except Exception as e:
```

Comment:
Should we also print/log the exception? If not, why do we explicitly "except" it?